### ISSUE TITLE
World Creation Crash 

### ISSUE THREAD

**Issue by splargtargafoonk** [ReikaKalseki/Reika_Mods_Issues#406]
It seems to be from Chromaticraft. I wanted to know if disabling chromaticraft and creating a new world, then restarting into that created world with chromaticraft will have any undesired effects (no chromaticraft content in the new world)

http://pastebin.com/GP9E9snZ

**Comment by ReikaKalseki** [comment]
Yes, it will make the mod unusable.

Also, your crash is caused by something - almost certainly another mod - calling world generators with a null world. This is a very severe (and blindingly obvious) error on their end.

**Comment by ljfa-ag** [comment]
Well actually it happened while a new World was constructed. The only other mod in the trace is Fastcraft, does it happen without it?

**Comment by lossycrypt** [comment]
Trying one of the newer fastcraft test builds might also help if that is the cause.

**Comment by ReikaKalseki** [comment]
A mod like FastCraft is one of the highest-level suspects, yes.

**Comment by splargtargafoonk** [comment]
It works after I downgraded my Forge

**Comment by ReikaKalseki** [comment]
I still suspect another mod is at fault.

**Comment by splargtargafoonk** [comment]
*Update: I have found a better solution to the problem that you might find useful @ReikaKalseki 

The conflict stems from biomes o plenty. The way I fixed it was to change the ids of Chromaticraft instead of BiomesOP. Changing Biomes and leaving Chromaticraft alone would not work. Really glad you added the option to change IDs! I'm aware of the need to record down whatever changes I make to your ID config, so now worries there

Not sure why downgrading the java previously worked though..it doesn't anymore
